scripts/regression/windows.ps1

```
@@ -0,0 +1,709 @@
+        Set-Content -Path $OutFile -Value ($stdoutText + $stderrText)
+        Remove-Item -Force -ErrorAction Ignore $stdoutFile, $stderrFile
+    }
+    else {
+        Set-Content -Path $OutFile -Value $stdoutText
+        Set-Content -Path $stderrFile -Value $stderrText
+    }
+    Remove-Item -Force -ErrorAction Ignore $scriptFile
+    return $exitCode
+}
+
+function Run-Cmd {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Run-Cmd' uses an unapproved verb.

(PSUseApprovedVerbs)